src/conditional-flow/multiple-conditions.js

```diff
@@ -28,9 +26,15 @@ function isHelloOrGoodbye (val1) {
 // 5-12   | Child
 // 13-19  | Teenager
 // 20+    | Adult
-function getAgeDescription (age) {
 
+function getAgeDescription (age) {
+ 
   // TODO: write code in this function body to pass the tests
+  return age === 0 ? 'Baby' : 
+         age >= 1 && age <= 4 ? 'Toddler' : 
```

Comment:
Nested ternaries are not a good idea to use generally. They're not easy to read, and harder to maintain than if/else statements. It's especially hard if you wanted to add a new case to match; for example if you wanted to add a new age description such as "Young adult"